docs/reference/compatibility.txt

```diff
@@ -491,3 +491,74 @@ are supported by Mongoid.
   8.0 and 8.1 stable branches.
 
 .. include:: /includes/unicode-checkmark.rst
+.. include:: /includes/unicode-ballot-x.rst
+
+Rails Frameworks Support
+------------------------
+
+Ruby on Rails is comprised of a number of frameworks, which Mongoid attempts to
+provide compatibility with wherever possible.
+
+Though Mongoid attempts to offer API compatibility with `Active Record <https://guides.rubyonrails.org/active_record_basics.html>`_,
+libraries that depend directly on Active Record may not work as expected when
+Mongoid is used as a drop-in replacement.
+
+.. note::
+
+  Mongoid can be used alongside Active Record within the same application without issue.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large no-padding
+
+   * - Rails Framework
+     - Supported?
+
+   * - ``ActionCable``
+     - |checkmark| [#rails-actioncable-dependency]_
+
+   * - ``ActionMailbox``
+     - |x| [#rails-activerecord-dependency]_
+
+   * - ``ActionMailer``
+     - |checkmark|
+
+   * - ``ActionPack``
+     - |checkmark|
+
+   * - ``ActionText``
+     - |x| [#rails-activerecord-dependency]_
+
+   * - ``ActionView``
+     - |checkmark|
+
+   * - ``ActiveJob``
+     - |checkmark| [#rails-activejob-dependency]_
+
+   * - ``ActiveModel``
+     - |checkmark|
+
```

Comment:
It's worth adding ActiveSupport to this framework list and mention that Mongoid requires ActiveSupport and uses it heavily. (This is obvious to us, but folks unfamiliar with Mongoid may not be aware.)